desktop/cmp/input/Select.ts

```diff
@@ -449,7 +460,7 @@ class SelectInputModel extends HoistInputModel {
         if (isNil(internal)) return null;
```

Comment:
shouldn't this also be `emptyValue`?   Also, I think we may need the *opposite* transform for an override to `toInternal`.